packages/trees/backend/sirius-components-collaborative-trees/src/main/java/org/eclipse/sirius/components/collaborative/trees/TreeEventProcessorFactory.java

```diff
@@ -41,6 +42,8 @@
 @Service
 public class TreeEventProcessorFactory implements IRepresentationEventProcessorFactory {
 
+    public static final String DETAILS_VIEW_ID = UUID.nameUUIDFromBytes("explorer_tree_description".getBytes()).toString();
```

Comment:
- This is not related to the details view, be more careful while copying code
- This is not related to the explorer, the explorer is provided by a class in Sirius Web named `ExplorerDescriptionProvider`
- You are copying the same thing in very different locations, just reference what's in `ExplorerDescriptionProvider`

Reply:
The explorer in ExplorerDescriptionProvider is using this ID. I can't do it the other way around.